src/Backend/Modules/Pages/Ajax/UploadFile.php

```diff
@@ -100,6 +106,9 @@ private function writeFile(string $content, string $fileName, string $destinatio
         // convert the filename to url friendly version
         $baseName = Uri::getUrl(pathinfo($fileName, PATHINFO_FILENAME));
         $extension = pathinfo($fileName, PATHINFO_EXTENSION);
+        if (!in_array(strtolower($extension), ['jpg', 'jpeg', 'gif', 'png'])) {
```

Comment:
That list is a bit restrictive.... Earlier this week I was using a user template block in Fork CMS and uploaded `.webp` and `.svg` files for example. `.avif` is a good one to have too in a year or so.

Reply:
I'll add those

Reply:
0117c1e4e6c180ed134653066d1196745b31a43a

Reply:
Awesome!